src/utils/logger.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+
+export class Log {
+  private output: vscode.LogOutputChannel;
+
+  constructor() {
+    this.output = vscode.window.createOutputChannel("Ansible", { log: true });
```

Comment:
@ganeshrn I see we already have an "Ansible Support" output channel. Should we use that for everything, or shall I go ahead with an "Ansible" one? Specifically here I'm logging the behavior around the two auth flavors.